cli/compose/loader/loader.go

```diff
@@ -100,7 +102,7 @@ func Load(configDetails types.ConfigDetails, options ...func(*Options)) (*types.
 			}
 		}
 
-		cfg, err := loadSections(configDict, configDetails)
+		cfg, err := loadSections(configDict, configDetails, *opts)
```

Comment:
Instead of passing the option through here, we could just as well discard the env-files here